scripts/lib/assemble_app.sh

```
@@ -61,6 +61,33 @@ assemble_app() {
+        # target). Homebrew dylibs are built for the builder's current SDK (e.g.
+        # macOS 26), which makes dyld refuse to load them -- and therefore the
+        # whole app, since they are on libaubio's launch-time load chain -- on any
+        # older system. Re-stamp each slice's LC_BUILD_VERSION down to 14.0; these
+        # are leaf codec/DSP libs that use no newer APIs, so this is safe. The
+        # final codesign pass in build_dmg/build_mas re-signs afterward.
+        restamp_min_os() {
+            local dylib="$1" target="14.0"
+            local archs; archs=$(lipo -archs "$dylib" 2>/dev/null); [[ -z "$archs" ]] && return 0
+            chmod u+w "$dylib" 2>/dev/null || true
+            local tmp; tmp=$(mktemp -d)
+            if [[ $(echo $archs | wc -w) -eq 1 ]]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)